packages/auth/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@recidiviz/auth",
+  "version": "1.0.0",
+  "description": "Auth0-Powered Authentication Component",
+  "author": "Recidiviz <team@recidiviz.org>",
+  "license": "GPL-3.0-only",
+  "main": "index.ts",
+  "scripts": {
+    "build": "rollup -c",
```

Comment:
once you get this working you may find it helpful to add a "watch" command similar to what we have in the design system also. Useful as a developer tool although not actually part of the publishing chain so not strictly necessary

Reply:
Ah, nice! Thank you!